lrpar/README.md

```diff
@@ -97,7 +97,7 @@ use lrpar::lrpar_mod;
 
 // Using `lrlex_mod!` brings the lexer for `calc.l` into scope.
 lrlex_mod!("calc.l");
-// Using `lrpar_mod!` brings the lexer for `calc.l` into scope.
+// Using `lrpar_mod!` brings the grammar for `calc.l` into scope.
```

Comment:
`calc.l` -> `calc.y`

Reply:
Also I think I called it `parser` in my changes, tho that might have been incorrect.

Reply:
Mea culpa! Fixed in https://github.com/softdevteam/grmtools/pull/137/commits/85c359e7af85e06cb7671639ec2700f84a4d759c.